apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.IntegrationTests/ModifyBucketLabelsTest.cs

```diff
@@ -269,10 +269,14 @@ public void ClearBucketLabels_NoLabelsBefore(bool runAsync)
             Assert.Empty(result);
         }
 
-        // TODO: Move these somewhere common, if this proves a useful pattern.
         private static T RunMaybeAsync<T>(bool runAsync, Func<T> sync, Func<Task<T>> async)
+        {
             // Run the async func separately to explicitly avoid synchronization etc.
-            => runAsync ? Task.Run(() => async().Result).Result : sync();
+            var result = runAsync? Task.Run(() => async().Result).Result : sync();
+            // Sleep, as everything in this test modifies buckets, and we can't do that very frequently.
+            StorageFixture.SleepAfterBucketCreateDelete();
```

Comment:
I think this will work as long as none of the tests do several updates themselves. I've taken a quick look at the tests, and maybe the `_RetrySucceeds` or `_RetryFails` might still hit quota issues? Not sure though.

Reply:
It's possible they'll still have problems - it's unclear. I *suspect* the quota is checked on an aggregate basis and that we're actually sleeping for longer than we need to, so this is "good enough". We can tweak it if we need to, of course. Will merge for now.